jumble/src/views/CharmDetailView.tsx

```diff
@@ -264,7 +264,18 @@ function useCodeEditor(
 
   const saveChanges = useCallback(() => {
     if (workingSrc && iframeRecipe && charm) {
-      saveNewRecipeVersion(charmManager, charm, workingSrc, iframeRecipe.spec);
+      generateNewRecipeVersion(
+        charmManager,
+        charm,
+        workingSrc,
+        iframeRecipe.spec,
+      ).then((newCharm) => {
+        console.log("Fixme, navigate to new charm", newCharm);
+        // navigate(createPath("charmShow", {
+        //   charmId: charmId(newCharm)!,
+        //   replicaName: replicaName,
+        // }));
+      });
```

Comment:
Seems like we should finish this?

Reply:
yes, this is the ticket around finishing code editting which is what I want to get to next